ssl/quic/quic_local.h

```diff
@@ -321,7 +321,7 @@ const SSL_METHOD *func_name(void)  \
                 NULL /* dispatch_alert */, \
                 ossl_quic_ctrl, \
                 ossl_quic_ctx_ctrl, \
-                NULL /* get_cipher_by_char */, \
+                ssl3_get_cipher_by_char, \
```

Comment:
I suppose this is needed to make SSL_CIPHER_find work.

However we only support a restricted set of ciphersuites for QUIC &mdash; specifically AES-128-GCM, AES-256-GCM and ChaCha20-Poly1305. Do we need to use a custom implementation that restricts what can be retrieved here?

Adding a test that e.g. CCM cannot be retrieved via SSL_CIPHER_find might be nice while we are here.